config/sets/symfony/symfony62.php

```diff
@@ -16,6 +23,53 @@
             'Sensio\Bundle\FrameworkExtraBundle\Configuration\Cache' => 'Symfony\Component\HttpKernel\Attribute\Cache',
             // @see https://github.com/symfony/symfony/pull/46906
             'Sensio\Bundle\FrameworkExtraBundle\Configuration\Template' => 'Symfony\Bridge\Twig\Attribute\Template',
+            // @see https://github.com/symfony/symfony/pull/46714
+            'Symfony\Component\Mailer\Bridge\OhMySmtp\Transport\OhMySmtpApiTransport' => 'Symfony\Component\Mailer\Bridge\MailPace\Transport\MailPaceApiTransport',
+            'Symfony\Component\Mailer\Bridge\OhMySmtp\Transport\OhMySmtpSmtpTransport' => 'Symfony\Component\Mailer\Bridge\MailPace\Transport\MailPaceSmtpTransport',
+            'Symfony\Component\Mailer\Bridge\OhMySmtp\Transport\OhMySmtpTransportFactory' => 'Symfony\Component\Mailer\Bridge\MailPace\Transport\MailPaceTransportFactory',
+            // @see https://github.com/symfony/symfony/pull/47363
+            'Symfony\Component\HttpKernel\Controller\ArgumentValueResolverInterface' => 'Symfony\Component\HttpKernel\Controller\ValueResolverInterface',
+            // @see https://github.com/symfony/symfony/pull/46094
+            'Symfony\Component\Security\Core\Security' => 'Symfony\Bundle\SecurityBundle\Security\Security',
+            // @see https://github.com/symfony/symfony/pull/46161
+            'Symfony\Component\Translation\Extractor\PhpAstExtractor' => 'Symfony\Component\Translation\Extractor\PhpAstExtractor',
+            // @see https://github.com/symfony/symfony/pull/47595
+            'Symfony\Component\HttpFoundation\ExpressionRequestMatcher' => 'Symfony\Component\HttpFoundation\RequestMatcher\ExpressionRequestMatcher',
+            'Symfony\Component\HttpFoundation\RequestMatcher' => 'Symfony\Component\HttpFoundation\RequestMatcher\ChainRequestMatcher',
         ],
     );
+
+    $rectorConfig->ruleWithConfiguration(
+        RenameMethodRector::class,
+        [
+            // @see https://github.com/symfony/symfony/pull/46854
+            new MethodCallRename(
+                'Symfony\Bundle\FrameworkBundle\Controller\AbstractController',
+                'renderForm',
+                'render'
+            ),
+            // @see https://github.com/symfony/symfony/pull/45034
+            new MethodCallRename(
+                'Symfony\Component\HttpFoundation\Request',
+                'getContentType',
+                'getContentTypeFormat'
+            ),
+            // @see https://github.com/symfony/symfony/pull/47711
+            new MethodCallRename('Symfony\Component\Mime\Email', 'attachPart', 'addPart'),
+            // @see https://github.com/symfony/symfony/pull/47363
+            new MethodCallRename(
```

Comment:
Is this allowed because return type is diffrent supports: bool resolve: array

Reply:
Per comment on https://github.com/rectorphp/rector-symfony/pull/283#discussion_r1029286911

this part may  need to be reverted